lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -16,21 +16,24 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     const tmpl = this.dom.cloneTemplate('#tmpl-lh-perf-metric', this.templateContext);
     const element = this.dom.find('.lh-perf-metric', tmpl);
     element.id = audit.result.name;
+    // FIXME(paulirish): currently this sets a 'lh-perf-metric--undefined' class on error'd audits
     element.classList.add(`lh-perf-metric--${Util.calculateRating(audit.result.score)}`);
 
     const titleEl = this.dom.find('.lh-perf-metric__title', tmpl);
     titleEl.textContent = audit.result.description;
 
-    const valueEl = this.dom.find('.lh-perf-metric__value span', tmpl);
+    const valueEl = this.dom.find('.lh-perf-metric__value', tmpl);
     valueEl.textContent = audit.result.displayValue;
 
     const descriptionEl = this.dom.find('.lh-perf-metric__description', tmpl);
     descriptionEl.appendChild(this.dom.convertMarkdownLinkSnippets(audit.result.helpText));
 
-    if (typeof audit.result.rawValue !== 'number') {
-      const debugStrEl = this.dom.createChildOf(element, 'div', 'lh-debug');
-      debugStrEl.textContent = audit.result.debugString || 'Report error: no metric information';
-      return element;
+    if (audit.result.error) {
+      element.classList.add(`lh-perf-metric--error`);
+      descriptionEl.textContent = '';
+      valueEl.textContent = 'Error';
```

Comment:
ha, well I actually liked the bangful message. Agree that `Error` just sitting on its own looks incomplete or like the string accidentally leaked into the html there rather than intentionally placed :)

Reply:
if it were small caps'd and had a hover state ala material design would you feel differently? :)

Reply:
ha, I think it's making it feel deliberate rather than any particular way of doing that :) Right now it kind of looks like we had a error string of `'Error\nsomething something'` and the rest of the string was cut off by the template or overflow hidden or something

Reply:
`¡Error!`

Reply:
> ¡Error!

this 💯